scripts/run.sh

```diff
@@ -81,23 +81,30 @@ function install() {
         then
             echo -e -n "${CYAN}(!)${NC} Enter your email address (Let's Encrypt will send you certificate expiration reminders): "
             read EMAIL
-            echo ""
-    
             mkdir -p $OUTPUT_DIR/letsencrypt
-            docker pull certbot/certbot
-            docker run -it --rm --name certbot -p 80:80 -v $OUTPUT_DIR/letsencrypt:/etc/letsencrypt/ certbot/certbot \
-                certonly --standalone --noninteractive  --agree-tos --preferred-challenges http \
-                --email $EMAIL -d $DOMAIN --logs-dir /etc/letsencrypt/logs
+            echo ""    
         fi
     fi
-    
+
     pullSetup
     docker run -it --rm --name setup -v $OUTPUT_DIR:/bitwarden \
         --env-file $ENV_DIR/uid.env bitwarden/setup:$COREVERSION \
         dotnet Setup.dll -install 1 -domain $DOMAIN -letsencrypt $LETS_ENCRYPT -os $OS \
         -corev $COREVERSION -webv $WEBVERSION
-    
     echo ""
+
+    if [ "$LETS_ENCRYPT" == "y" ]
```

Comment:
Because before setup we don't have user configured port numbers yet, so we don't know how to properly run Let's Encrypt.
This issue was also reported in #271.
Whenever certification creation fails before or after setup, user will have to delete its `bwdata` dir to retry installation process. So it does not change too much things.
Running Let's Encrypt after setup gives less chances to fail, as correct ports will be used.